contracts/deprecated/RelayerIncentivize.sol

```diff
@@ -126,72 +80,6 @@ contract RelayerIncentivize is IRelayerIncentivize, System, IParamSubscriber {
-        uint256[] memory relayerWeight = new uint256[](relayers.length);
-        for (uint256 index = 0; index < relayers.length; ++index) {
-            address relayer = relayers[index];
-            uint256 weight = calculateHeaderRelayerWeight(headerRelayersSubmitCount[relayer]);
-            relayerWeight[index] = weight;
-            totalWeight = totalWeight.add(weight);
-        }
-
-        uint256 callerReward = totalReward.mul(callerCompensationMolecule).div(callerCompensationDenominator);
-        totalReward = totalReward.sub(callerReward);
-        uint256 remainReward = totalReward;
-        for (uint256 index = 1; index < relayers.length; ++index) {
-            uint256 reward = relayerWeight[index].mul(totalReward).div(totalWeight);
-            relayerRewardVault[relayers[index]] = relayerRewardVault[relayers[index]].add(reward);
-            remainReward = remainReward.sub(reward);
-        }
-        relayerRewardVault[relayers[0]] = relayerRewardVault[relayers[0]].add(remainReward);
-
-        collectedRewardForHeaderRelayer = 0;
-        for (uint256 index = 0; index < relayers.length; ++index) {
-            delete headerRelayersSubmitCount[relayers[index]];
-        }
-        delete headerRelayerAddressRecord;
-        return callerReward;
-    }
-
-    function distributePackageRelayerReward() internal returns (uint256) {
-        uint256 totalReward = collectedRewardForTransferRelayer;
-
-        uint256 totalWeight = 0;
-        address payable[] memory relayers = packageRelayerAddressRecord;
-        uint256[] memory relayerWeight = new uint256[](relayers.length);
-        for (uint256 index = 0; index < relayers.length; ++index) {
-            address relayer = relayers[index];
-            uint256 weight = calculateTransferRelayerWeight(packageRelayersSubmitCount[relayer]);
-            relayerWeight[index] = weight;
-            totalWeight = totalWeight + weight;
-        }
-
-        uint256 callerReward = totalReward.mul(callerCompensationMolecule).div(callerCompensationDenominator);
-        totalReward = totalReward.sub(callerReward);
-        uint256 remainReward = totalReward;
-        for (uint256 index = 1; index < relayers.length; ++index) {
-            uint256 reward = relayerWeight[index].mul(totalReward).div(totalWeight);
-            relayerRewardVault[relayers[index]] = relayerRewardVault[relayers[index]].add(reward);
-            remainReward = remainReward.sub(reward);
-        }
-        relayerRewardVault[relayers[0]] = relayerRewardVault[relayers[0]].add(remainReward);
-
-        collectedRewardForTransferRelayer = 0;
-        for (uint256 index = 0; index < relayers.length; ++index) {
-            delete packageRelayersSubmitCount[relayers[index]];
-        }
-        delete packageRelayerAddressRecord;
-        return callerReward;
-    }
-
     function calculateTransferRelayerWeight(uint256 count) public pure returns (uint256) {
```

Comment:
no one is using `calculateTransferRelayerWeight`

Reply:
removed